gabrielaEnache/src/main/java/com/endava/store/User.java

```diff
@@ -0,0 +1,14 @@
+package com.endava.store;
+
+
+public class User extends AbstractUser {
+
+    public User(String userName) {
+        super(userName);
+    }
+
+    @Override
+    public String toString() {
+        return super.toString();
```

Comment:
if you don't add any new logic to method, there is no need to override it

Reply:
Fixed